msg/GenericTrajectory.msg

```diff
@@ -0,0 +1,3 @@
+Header header
+trajectory_msgs/JointTrajectory joint_trajectory
```

Comment:
I agree with @v4hn's comment [here](https://github.com/ros-planning/moveit_msgs/pull/46/files#r313735106), there probably doesn't need to be an explicit field choosing which trajectory, but I'd at least add a comment saying that the unintended trajectory will be empty.